travis.sh

```diff
@@ -123,7 +123,9 @@ travis_time_start setup_catkin
 sudo apt-get install -y -q -qq ros-$ROS_DISTRO-roslaunch
 (cd /opt/ros/$ROS_DISTRO/lib/python2.7/dist-packages; wget --no-check-certificate https://patch-diff.githubusercontent.com/raw/ros/ros_comm/pull/641.diff -O /tmp/641.diff; [ "$ROS_DISTRO" == "hydro" ] && sed -i s@items@iteritems@ /tmp/641.diff ; sudo patch -p4 < /tmp/641.diff)
 
-travis_time_end
+travis_time_end  # setup_catkin
+
+travis_time_start check_version_ros
```

Comment:
travis_time_start function folds the result of enclosed section, which I think makes the result much more searchable. I'm applying this for the most of sections.